pkg/cache/postgres.go

```diff
@@ -34,10 +34,22 @@ func (c *postgresCache) Add(check pkg.Check, statii ...pkg.CheckStatus) {
 		} else {
 			check.Status = "unhealthy"
 		}
+		c.AddCheckFromStatus(check, status)
 		c.AddCheckStatus(check, status)
 	}
 }
 
+func (c *postgresCache) AddCheckFromStatus(check pkg.Check, status pkg.CheckStatus) {
+	if status.Check == nil {
+		return
+	}
+	// Before syncing canary, mark all these checks as deleted_at
+	check.Labels["transformed-from"] = check.CanaryID.String()
```

Comment:
Is this always set even for non-transformed values ? It might be better to make this an actual column rather than a label

Reply:
Have fixed the always set option, if you want I can make it a column.

Reply:
We probably also need to change it to not delete the check, but mark it as healthy / complete  - For alertmanager it might return the status as not firing and the time it stopped firing and this code would then only be used as a backup